packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -276,48 +247,183 @@ function ListView() {
   }
 
   return (
-    <Wrapper>
+    <Layout>
       <PageTitle name="Webhooks" />
-      <Header {...headerProps} />
-      {canRead && (
-        <div className="list-wrapper">
+      <HeaderLayout
+        title={headerProps.title.label}
+        subtitle={headerProps.content}
+        primaryAction={(
+          <Button onClick={() => (canCreate ? handleGoTo('create') : {})} startIcon={<AddIcon />}>
+            {headerProps.addBtnLabel}
+          </Button>
+        )}
+      />
+
+      <ContentLayout>
+        <>
           {rowsCount > 0 ? (
-            <List
-              {...listProps}
-              customRowComponent={props => {
-                return (
-                  <ListRow
-                    {...props}
-                    canUpdate={canUpdate}
-                    canDelete={canDelete}
-                    onCheckChange={handleChange}
-                    onEditClick={handleGoTo}
-                    onDeleteCLick={handleDeleteClick}
-                    onEnabledChange={handleEnabledChange}
-                    itemsToDelete={webhooksToDelete}
-                  />
-                );
-              }}
-            />
+            <Table
+              colCount={5}
+              rowCount={webhooks.length}
+              footer={(
+                <TFooter onClick={() => (canCreate ? handleGoTo('create') : {})} icon={<AddIcon />}>
+                  Add another field to this collection type
```

Comment:
I think this should go in a translation file 😊 . You have exampelp in this file using the `formatMessage` function